R/personnel-element.R

```diff
@@ -28,32 +28,28 @@
 #'               organization = 'IBM')
 #' @export
 add_personnel <- function(parent_element, first_name, last_name, email, 
-                          role, orcid = NULL, organization = NULL) {
+                          role, organization, orcid = NULL) {
```

Comment:
the params in the docs are not in the same order as in the function signature, plz fix

Reply:
Will do.